pkg/resources/network/network.go

```diff
@@ -0,0 +1,13 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+// Package network provides resources which describe networking subsystem state.
+package network
+
+import (
+	"github.com/cosi-project/runtime/pkg/resource"
+)
+
+// NamespaceName contains resources related to networking.
+const NamespaceName resource.Namespace = "net"
```

Comment:
In other naming decisions we have stuck with more canonical names. Consider "network" instead of "net".

Reply:
yep, agreed, I will get it updated :+1: